worlds/yoshisisland/Rules.py

```diff
@@ -0,0 +1,597 @@
+from .level_logic import YoshiLogic
+from worlds.generic.Rules import set_rule
+
+
+
+logic: YoshiLogic
+def set_easy_rules(world, multiworld, player: int, boss_order, luigi_pieces):

```

Comment:
Recommend reworking set_easy_rules to only take world, since multiworld and player you can get from world, and you aren't using boss_order or luigi_pieces